inspirehep/modules/hal/tei.py

```diff
@@ -122,7 +122,7 @@ def _parse_structures(record):
                 continue
 
     try:
-        records = get_es_records('ins', recids)
+        records = get_es_records('recid', recids)
```

Comment:
Yes, the `recid` will be implicit and people don't have to think about it.

On the other hand, you will have to pass a more high level endpoint in order to know what search class to use (and what index). So this call would be:

```
records = get_es_records('literature', recids)
```

Where literature is the key in `RECORDS_REST_ENDPOINTS`